homu/main.py

```diff
@@ -450,14 +450,16 @@ class LabelEvent(Enum):
 PORTAL_TURRET_IMAGE = "https://cloud.githubusercontent.com/assets/1617736/22222924/c07b2a1c-e16d-11e6-91b3-ac659550585c.png"  # noqa
 
 
-def parse_commands(body, username, repo_label, repo_cfg, state, my_username,
-                   db, states, *, realtime=False, sha='', command_src=''):
+def parse_commands(body, username, user_id, repo_label, repo_cfg, state,
```

Comment:
This signature is getting awfully wordy. Would it make sense to wrap up `body`, `username`, and `user_id` into some sort of object that represents a comment?

Maybe that's out of the scope of the PR, though?

Reply:
I think it would be fine to improve this in another PR.